Sources/Receipt/ReceiptDetailSectionData.swift

```diff
@@ -113,10 +113,13 @@ struct ReceiptDetailSectionFeeData: ReceiptDetailSectionData {
                                      field: "amount"))
         var fee: Double = 0.0
         if let strFee = receipt.fee {
-            rows.append(ReceiptDetailRow(title: "receipt_details_fee".localized(),
-                                         value: String(format: valueCurrencyFormat, strFee, receipt.currency),
-                                         field: "fee"))
             fee = Double(strFee) ?? 0.0
+
+            if fee != 0.0 {
```

Comment:
The ticket is to not display the entire section if there's no fee. So add an if check in Presenter:
`if let receiptFeeSection = ReceiptDetailSectionFeeData(from: receipt) {
            sectionData.append(receiptFeeSection)
        }`

and update ReceiptDetailSectionFeeData to initialize only if there's fee else return nil.

Reply:
Thanks, updated